src/utils/utils.py

```diff
@@ -10,7 +10,6 @@
 import logging
 import codecs
 import time
-
```

Comment:
Revert.

Reply:
This wasn't me. This is also from load_tsv.